Queue/Queue.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Algorythim analisys of Queues and it's basic operations in JavaScript
+ * Goal: Improve knowledge about queue data structures and improve practice
+ *
+ * Queue implementation through Nodes
+ */
+
+// Creates a Node with the data informed and sets the Next attr
+class Node {
+  constructor(data) {
+    (this.data = data), (this.next = null);
+  }
+}
+
+class Queue {
+  // Sets the front and back attrs for the queue
+  constructor() {
+    this.front = null;
+    this.back = null;
+  }
+
+  // Adds/Stores data as a Node in the back of the Queue
+  enqueue = data => {
+    var newNode = new Node(data);
+    if (this.back) {
+      this.back.next = newNode;
```

Comment:
this.back.next? isso ta correto?

Reply:
Minha ideia é que o ultimo cara da fila (next = null) tenha agora um next que é o newNode e ele entao passa a ser novo ultimo cara